src/main/java/org/apache/sling/commons/threads/impl/DefaultThreadPoolManager.java

```diff
@@ -320,6 +330,9 @@ public void deleted(String pid) {
 
         private ServiceRegistration mbeanRegistration;
 
+        @SuppressWarnings("rawtypes")
```

Comment:
Why do you need this warning here?

Reply:
@rombert As Gauge is parametrized class, and making it here Gauge<?> complicates things a bit further (casting etc).

Reply:
Ah, ok.